mmv1/products/alloydb/product.yaml

```diff
@@ -11,8 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
----
```

Comment:
looks like it doesn't need to be on the same line. I think either way is fine, as long as we can enforce consistency with the linter.